be/src/storage/lake/delta_writer.cpp

```diff
@@ -287,6 +322,22 @@ Status DeltaWriterImpl::handle_partial_update() {
         }
         _tablet_schema = _partial_update_tablet_schema;
     }
+
+    auto sort_key_idxes = _tablet_schema->sort_key_idxes();
+    std::sort(sort_key_idxes.begin(), sort_key_idxes.end());
+    bool auto_increment_in_sort_key = false;
+    for (auto& idx : sort_key_idxes) {
+        auto& col = _tablet_schema->column(idx);
+        if (col.is_auto_increment()) {
+            auto_increment_in_sort_key = true;
+            break;
+        }
+    }
+
+    if (auto_increment_in_sort_key && _miss_auto_increment_column) {
+        LOG(WARNING) << "table with sort key do not support partial update";
+        return Status::NotSupported("table with sort key do not support partial update");
```

Comment:
The error msg should be "auto increment column ins sort key do not support partial update" ? or other?

Reply:
done